cmd/commands/commands.go

```diff
@@ -78,7 +78,9 @@ func replaceCustomData(jsonBytes []byte) ([]byte, error) {
 	var buf bytes.Buffer
 	err := json.Indent(&buf, replacedBytes, "", "    ")
 	if err != nil {
-		return nil, err
+		// If we can't indent the JSON, it likely means the replacement
+		// data wasn't correct, so we return the original JSON.
+		return jsonBytes, nil
```

Comment:
Reviewer note: the jsonBytes with error context is decorated here:
https://github.com/lightningnetwork/lnd/blob/19653e1deca8baaa2c7f9bff2cef334dfc081eb9/cmd/commands/commands.go#L123C1-L126C32

Reply:
Hmm, good point. Updated.